packages/rpc/src/common.ts

```diff
@@ -104,10 +104,10 @@ export type RpcRequest = WithHash<{
     destinationAsset: UncheckedAssetAndChain,
     destinationAddress: string,
     brokerCommission: number,
-    ccmMetadata?: Nullish<CcmParams>,
-    boostFee?: Nullish<number>,
-    affiliateFees?: Nullish<{ account: string; bps: number }[]>,
-    fillOrKillParams?: FillOrKillParams,
+    ccmMetadata: Nullish<CcmParams>,
+    boostFee: Nullish<number>,
+    affiliateFees: Nullish<{ account: string; bps: number }[]>,
```

Comment:
fillOrKillParams is not optional anymore and you cannot have optional params before mandatory params